scripts/generate-version-info.sh

```diff
@@ -16,8 +16,17 @@ OUTPUT_FILE="$1"
 sudo rpm --query --all --queryformat '\{"%{NAME}": "%{VERSION}-%{RELEASE}"\}\n' | jq --slurp --sort-keys 'add | {packages:(.)}' > "$OUTPUT_FILE"
 
 # binaries
-echo $(jq ".binaries.kubelet = \"$(kubelet --version | awk '{print $2}')\"" $OUTPUT_FILE) > $OUTPUT_FILE
-echo $(jq ".binaries.awscli = \"$(aws --version | awk '{print $1}' | cut -d '/' -f 2)\"" $OUTPUT_FILE) > $OUTPUT_FILE
+KUBELET_VERSION=$(kubelet --version | awk '{print $2}')
+if [ "$?" != 0 ]; then
+  echo "unable to get kubelet version"
+fi
+echo $(jq ".binaries.kubelet = \"$KUBELET_VERSION\"" $OUTPUT_FILE) > $OUTPUT_FILE
```

Comment:
Isn't this just failing in the same way, but with a different log?

Reply:
currently it is not failing when `kubelet --version` fails as it is earlier in the pipe (even though we set `pipefail`), the `errexit` does not kick in either. so we have to check this explicitly